uwb-beacon-firmware/CMakeLists.txt

```diff
@@ -0,0 +1,158 @@
+add_library(state_estimation
+    src/state_estimation.cpp
+)
+
+target_include_directories(state_estimation PUBLIC src)
+
+target_link_libraries(state_estimation
+    Eigen
+)
+
+cvra_add_test(TARGET ekf_test SOURCES
+    tests/ekf.cpp
+    tests/state_estimation.cpp
+    DEPENDENCIES
+    Eigen
+    state_estimation
+)
+
+add_library(lru_cache
+    src/lru_cache.c
+)
+
+target_include_directories(lru_cache PUBLIC src)
+
+cvra_add_test(TARGET lru_cache_test SOURCES
+    tests/lru_cache.cpp
+    DEPENDENCIES
+    lru_cache
+)
+
+add_library(mpu9250
+    src/mpu9250.c
+)
+
+target_include_directories(mpu9250 PUBLIC src)
```

Comment:
Is it worth adding separate library targets for sources that are not (yet) used anywhere else?

Reply:
I am not sure what you mean. That target is used in a test and in a binary.

Reply:
Ok, I missed that there was a unit test for it.